kafka-bare-quickstart/pom.xml

```diff
@@ -46,6 +46,12 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-kafka-client</artifactId>
     </dependency>
+    <dependency>
+      <!-- Workaround for https://github.com/quarkusio/quarkus/issues/22000 TODO: should be removed, when quarkus/hibernate team will fix the issue -->
```

Comment:
just say quarkus, hibernate is unrelated to that issue.

Reply:
Ooops, this was unintentional :D Fixed.